docs/porting-a-v2-ea-to-v3.md

```diff
@@ -81,36 +73,39 @@ export type EndpointTypes = {
   Provider: {
     ... // Provider specific details, these differ and are defined by each Transport implementation
   }
-  }
 }
 ```
 
 These types will also be shared with the Endpoint that’s referencing this transport.
 
-## Building a REST/Batch transport
+## Building an Http Transport
 
-Building a REST mainly consists of defining types (above), and defining how to build the request and parse the response
-to and from the DP.
+Building an HTTP transport mainly consists of defining types (above), and defining how to build the request and parse the response
+to and from the DP. An example that sends request to a non-batch endpoint:
 
 ```typescript
-const restTransport = new RestTransport<EndpointTypes>({
-  prepareRequest: (req, config) => {
-    // The `req` param contains the request made to the EA.
+const httpTransport = new HttpTransport<EndpointTypes>({
+  prepareRequests: (params, config) => {
+    // The `params` param contains the all the requests made to the EA that need data fetched from the DP.
     // Using this, return the request config to the DP.
-    return {
-      baseURL: config.API_ENDPOINT || DEFAULT_API_ENDPOINT,
-      url: '/price',
-      method: 'GET',
-      params: {
-        from: req.base,
-        to: req.quote,
-      },
-    }
+    return params.map(req => ({
+      params: req,
+      request: {
+        baseURL: config.API_ENDPOINT || DEFAULT_API_ENDPOINT,
```

Comment:
Probably something that we should have updated some time back but now that we've moved `API_ENDPOINT` to the EA side, don't we expect `config.API_ENDPOINT` to already have the default defined in it?
```suggestion
        baseURL: config.API_ENDPOINT,
```